pkgs/applications/version-management/git-and-tools/gitstatus/default.nix

```diff
@@ -0,0 +1,28 @@
+{callPackage, stdenv, fetchFromGitHub, ...}:
+
+stdenv.mkDerivation rec {
+  pname = "gitstatus";
+  version = "20190506";
+
+  src = fetchFromGitHub {
+    owner = "romkatv";
+    repo = "gitstatus";
+    rev = "9c791f93c23c04dadfab8b4309a863b62a6ee424";
+    sha256 = "0jbdrgl62x6j920h72n2q6304fb6gdgnmllpv4aa76m13b9qhgq6";
+  };
+
+  buildInputs = [ (callPackage ./romkatv_libgit2.nix {}) ];
```

Comment:
Not sure another file is needed. This derivation could be part of the current file in a `let` clause for instance.
But this is just  a remark (and not required).

Reply:
ack. separate file makes it easier for me to manage the rev/sha256 for github repo packages with a script I've got.